examples/slash_commands/option_choice.py

```diff
@@ -0,0 +1,43 @@
+from disnake import Option, OptionChoice, Interaction, Intents
+from disnake.ext import commands
+from disnake.ext.commands import slash_command
+
+import random
+
+intents = Intents.all()
+# In production please choose the intents coresponding to your use case.
+
+# We create the instance of the bot.
+bot = commands.Bot(intents=intents)
+
+# For this example we're going to make a rock-paper-scissors game
+# which will use OptionChoice as the input.
+@slash_command(name='rps', description='Play a game of rock-paper-scissors with the bot.',
+    options=[
+        Option(name='choice', description='Select your choice for rock-paper-scissors.', choices=[
+            OptionChoice(name='Rock', value='rock'),
+            OptionChoice(name='Paper', value='paper'),
+            OptionChoice(name='Scissors', value='scissors')
+        ],
+        required=True)  # We're going to make this be a required option.
+    ])
+async def rps(inter: Interaction, choice):  # NOTICE: The param which defines the option must have the exact same name.
```

Comment:
Here it should be `ApplicationCommandInteraction` instead of `Interaction` since it's a more relevant subclass.